pkgs/tools/system/pciutils/default.nix

```diff
@@ -30,8 +31,13 @@ stdenv.mkDerivation rec {
 
   installTargets = [ "install" "install-lib" ];
 
-  # Get rid of update-pciids as it won't work.
-  postInstall = "rm $out/sbin/update-pciids $out/man/man8/update-pciids.8";
+  postInstall = ''
+    # Get rid of update-pciids as it won't work ...
+    rm $out/sbin/update-pciids $out/man/man8/update-pciids.8
+
+    # ... but also update database with newer version from hwdata
+    cp --reflink=auto ${hwdata}/share/hwdata/pci.ids $out/share/pci.ids
```

Comment:
```suggestion
    ln -s ${hwdata}/share/hwdata/pci.ids $out/share/pci.ids
```

Reply:
But then pciutils will depend on the full closure of hwdata, right? We wanted to avoid that over at #112644. I'll add a comment to the source!